src/cryptonotecore/Core.cpp

```diff
@@ -1706,10 +1711,10 @@ namespace CryptoNote
         {
             logger(Logging::WARNING) << "Transaction " << cachedTransaction.getTransactionHash()
                                      << " is not valid. Reason: fee is too small and it's not a fusion transaction";
-            return false;
+            return {false, "Transaction fee is too small for the transaction"};
```

Comment:
```suggestion
            return {false, "Transaction fee is too small"};
```
Maybe :man_shrugging: